README.md

```diff
@@ -6,73 +6,72 @@ https://firecloud.dsde-dev.broadinstitute.org/
 
 https://firecloud.dsde-staging.broadinstitute.org/
 
-## Getting Started
+## Technologies
 
-This project runs in a Docker container. Docker is available here:
+[ClojureScript](https://github.com/clojure/clojurescript) is used for the UI.
 
-https://www.docker.com/
+We use the [Leiningen](http://leiningen.org/) build tool to manage ClojureScript dependencies.
 
-[ClojureScript](https://github.com/clojure/clojurescript) is used for the UI.
-We use the [Leiningen](http://leiningen.org/) build tool.
 The code incorporates usage of [react-cljs](https://github.com/dmohs/react-cljs) which is 
 a ClojureScript wrapper for [React](https://facebook.github.io/react/).
 
-For authentication, you will need to create a Google web application client ID from here:
+Figwheel replaces the running JavaScript within the page so changes are visible without a browser reload. More information [here](https://github.com/bhauman/lein-figwheel). [This video](https://www.youtube.com/watch?v=j-kj2qwJa_E) gives some insight into the productivity gains available when using this technology (up to about 15:00 is all that is necessary).
+
+## Getting Started
+
+Start with a [docker](https://www.docker.com/) environment.
+
+You will need to create a Google web application client ID from here:
```

Comment:
Added some detail here. I don't want to be too specific or else this will become wrong when Google updates their UI.